concordium-consensus/tools/database-exporter/README.md

```diff
@@ -0,0 +1,27 @@
+# database-exporter
```

Comment:
There is already a section in the README in `concordium-cosnensus/README.md` That should be removed in light of this, and just a link to this document tshould be added.